app/views/certificates.html

```diff
@@ -11,7 +11,7 @@
               Manage the OpenHIM keystore. A new OpenHIM server certificate can be added or generated and trusted client certificates can also be added or edited. Only self signed client certificates that should be trusted need to be added to the trust store. The OpenHIM already trusts all certificates signed by the common CAs, although, certificates must be linked to a client on the clients page before they can be used to send in requests.
             </p>
           </div>
-          <button class="btn btn-primary pull-right" ng-click="addCert('server')" ng-show="!alerts.server"><i class="glyphicon glyphicon-plus"></i> Create Server Certificate</button>
+          <button class="btn btn-primary pull-right" ng-click="addCert('server')" ng-if="currentServerCert && currentServerCert.watchFSForCert == false" ng-show="!alerts.server"><i class="glyphicon glyphicon-plus"></i> Create Server Certificate</button>
```

Comment:
I think we should add a note if watchFSFor Cert is enabled so say certs are controlled via the filesystem.